subprojects/gradle/build-verdict/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+    id("kotlin")
+    id("java-gradle-plugin")
+    `maven-publish`
+    id("com.jfrog.bintray")
+}
+
+dependencies {
+    implementation(gradleApi())
+    implementation(Dependencies.gradle.kotlinPlugin)
+    implementation(Dependencies.gradle.androidPlugin) {
+        because("without error occurs in tests")
```

Comment:
you can remove this line now? after #586 merged?

Reply:
#586 fixes not all scenarios. For example, if you depend on `kotlin-android` it will work:
- when your module doesn't have any of kotlin, android plugins
- when your module has both kotlin and android plugins

Reply:
in this case, it will be better to clarify `because` message: 
something like: `TestProjectGenerator depends on agp in plugin's classpath` or even link to documentation about this case (better one)

Reply:
http://links.k.avito.ru/mbs-9695